events/wc_offmap_events.txt

```diff
@@ -706,34 +882,614 @@ narrative_event = {
+							}
+						}
+					}
+				}
+				AND = {
+					title = c_sagehold
+					location = {
+						wonder = {
+							OR = {
+								has_wonder_upgrade = upgrade_old_god_nzoth_1
+								has_wonder_upgrade = upgrade_old_god_nzoth_2
+							}
+						}
+					}
+				}
+				AND = {
+					title = c_ulduar
+					location = {
+						wonder = {
+							OR = {
+								has_wonder_upgrade = upgrade_old_god_yogg_saron_1
+								has_wonder_upgrade = upgrade_old_god_yogg_saron_2
+							}
+						}
+					}
+				}
+			}
 		}
 	}
 	
 	option = {
 		name = EVTOPTA_WCOME.250
-		e_herald_cthun = {
-			owner = {
-				grant_title_no_opinion = c_ahnqiraj
+		if = {
+			limit = {
+				FROM = {
+					title = c_ahnqiraj
+					location = {
+						wonder = {
+							OR = {
+								has_wonder_upgrade = upgrade_old_god_cthun_1
+								has_wonder_upgrade = upgrade_old_god_cthun_2
+							}
+						}
+					}
+				}
+			}
+			e_herald_cthun = {
+				owner = {
+					grant_title_no_opinion = c_ahnqiraj
+				}
+			}
+		}
+		else_if = {
+			limit = {
+				FROM = {
+					title = c_sagehold
+					location = {
+						wonder = {
+							OR = {
+								has_wonder_upgrade = upgrade_old_god_nzoth_1
+								has_wonder_upgrade = upgrade_old_god_nzoth_2
+							}
+						}
+					}
+				}
+			}
+			e_herald_nzoth = {
+				owner = {
+					grant_title_no_opinion = c_sagehold
+				}
+			}
+		}
+		else = {
+			limit = {
+				FROM = {
+					title = c_ulduar
+					location = {
+						wonder = {
+							OR = {
+								has_wonder_upgrade = upgrade_old_god_yogg_saron_1
+								has_wonder_upgrade = upgrade_old_god_yogg_saron_2
+							}
+						}
+					}
+				}
+			}
+			e_herald_yoggsaron = {
+				owner = {
+					grant_title_no_opinion = c_ulduar
+				}
 			}
 		}
 	}
 }
+
+character_event = {
+	id = WCOME.300
+	desc = {
+       trigger = { ROOT = { has_character_modifier = follower_of_cthun } }
+       text = EVTDESC_WCOME.300_cthun
+       picture = GFX_evt_cthun
+    }
+	desc = {
+       trigger = { ROOT = { has_character_modifier = follower_of_yogg_saron } }
+       text = EVTDESC_WCOME.300_yoggsaron
+       picture = GFX_evt_yogg_saron
+    }
+	desc = {
+       trigger = { ROOT = { has_character_modifier = follower_of_nzoth } }
+       text = EVTDESC_WCOME.300_nzoth
+       picture = GFX_evt_nzoth
+    }
+	border = GFX_event_normal_frame_religion
+	
+	is_triggered_only = yes
+	
+	only_playable = yes
+	
+	trigger = {
+		religion = old_gods_worship
+	}
+	
+	option = {
+		name = EVTOPTA_WCOME.300
+		if = {
+			limit = { cthun_offmap_decisions_enabled_trigger = yes }
+			if = {
+				limit = { offmap_cthun = { has_status = old_god_imprisoned } }
+				offmap_cthun = { set_status = old_god_partly_imprisoned }
+				clr_character_flag = freeing_old_god_flag
+				break = yes
+			}
+			if = {
+				limit = { offmap_cthun = { has_status = old_god_partly_imprisoned } }
+				offmap_cthun = { set_status = old_god_partly_free }
+				set_global_flag = reawakening_of_cthun_flag
```

Comment:
So, the flag is set as a global one.
Never checked.
And cleared as a character one.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/87879819-0d493880-c9fe-11ea-886a-7945202ec74d.png)

</details>

Reply:
the global flag is used to enable decisions for qiraji

Reply:
Not addressed.

> the global flag is used to enable decisions, wars, etc for qiraji

Could you show where?

Reply:
can_declare_wars_trigger = {
	NOR = {
		is_untouchable_trigger = yes
		AND = {
			trait = creature_qiraji
			NOT = { has_global_flag = reawakening_of_cthun_flag }
		}
	}
}

Reply:
Why it's tied to the flag and not C'thun status?